core/drivers/ffa_console.c

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+ */
+
+#include <console.h>
+#include <drivers/serial.h>
```

Comment:
Add `#include <compiler.h>` (for `__unused` see below)

Reply:
Done